apps/indexer/src/api/controller/proposals-activity.controller.ts

```diff
@@ -47,6 +47,17 @@ export function proposalsActivity(
             .max(100, "Limit cannot exceed 100")
             .default(10)
             .optional(),
+          orderBy: z
+            .enum(["votingPower", "voteTiming"])
+            .default("voteTiming")
+            .optional(),
+          orderDirection: z.enum(["asc", "desc"]).default("desc").optional(),
+          userVoteFilter: z
+            .enum(["yes", "no", "abstain", "no-vote"])
```

Comment:
couldn't that come from [here](https://github.com/blockful/anticapture/pull/1057/files#diff-5d123be3614fa174dc4e143b28524749c9a95a2b546e99a5a6511fb3817ef346R17)?